perllib/FixMyStreet/Cobrand/Buckinghamshire.pm

```diff
@@ -144,6 +144,18 @@ sub report_new_munge_before_insert {
     $report->category('Flytipping (off-road)');
 }
 
+sub post_report_sent {
+    my ($self, $report) = @_;
+
+    foreach my $body_id ($report->bodies_str_ids) {
```

Comment:
This also needs `@{...}` around the arrayref, otherwise this loop is always called only once with $body_id set to the ref, not an ID.